### PR TITLE
feat: customize smtp server

### PR DESCRIPTION
**Pull request by sguiheux**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -116,8 +116,10 @@ def sendmail(self):
             msg['Subject'] = self.subject
             msg['From'] = self.fromm
             msg['To'] = self.mail_to
-
-            smtp = smtplib.SMTP('localhost')
+            smtpServer = 'localhost'
```
Comment (trnsnt) [inline bot.py]: smtpServer => smtp_server

File: bot.py [hunk 2]
```diff
@@ -116,8 +116,10 @@ def sendmail(self):
             msg['Subject'] = self.subject
             msg['From'] = self.fromm
             msg['To'] = self.mail_to
-
-            smtp = smtplib.SMTP('localhost')
```
Comment (trnsnt) [inline bot.py:120]: more pythonic to do
smtp_server = os.environ.get('SMTP_SERVER', 'localhost')
 :)

Comment (arnaudmorin) [reply bot.py:120]: agree

**Comment by AdrienR** [comment]
Too old PR

**Pull request closed without merging**